pages/nft-book-store/new.vue

```diff
@@ -160,6 +173,10 @@ function addNotificationEmail () {
   notificationEmailInput.value = ''
 }
 
+function onStripeConnectWalletInput () {
+  stripeConnectWallet.value = stripeConnectWalletInput.value.trim()
```

Comment:
use `v-model.trim`?

Reply:
the trim() is extra, the main goal here is to force stripeConnectWallet to update
seems combo-ing v-model on radio input (`stripeConnectWallet`) with a value which has v-model on text input (`stripeConnectWalletInput`) doesn't make `stripeConnectWallet` automatically update on input...
Maybe I add a comment